apps/src/templates/sectionAssessments/MultipleChoiceDetailsDialog.jsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable react/no-danger */
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import Button from '@cdo/apps/templates/Button';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import i18n from "@cdo/locale";
+import DialogFooter from "@cdo/apps/templates/teacherDashboard/DialogFooter";
+import processMarkdown from 'marked';
+import renderer from "@cdo/apps/util/StylelessRenderer";
+import {getCurrentQuestion} from "./sectionAssessmentsRedux";
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  },
+  instructions: {
+    marginTop: 20
+  }
+};
+
+class MultipleChoiceDetailsDialog extends Component {
```

Comment:
This is currently very similar to the one for free response questions, but they are about to diverge significantly so I went ahead and made them separate.

Reply:
thanks for the heads up.